wizard_agent/my_agent.py

```diff
@@ -45,7 +46,7 @@ def next_move(self, game_state, player_state):
             can_do_reload = self.strategies["reload"].can_execute(game_state, player_state)
             can_do_treasure = self.strategies["treasure"].can_execute(game_state, player_state)
             can_do_kill = self.strategies["kill"].can_execute(game_state, player_state)
-
+            can_do_collect = self.strategies["smartcollect"].can_execute(game_state,player_state)
             # Determine next action
             if can_do_treasure:
```

Comment:
can remove the treasure and reload strategy. 

if can_do_treasure and can_do_reload is compressed into can_do_collect